Big-Spender/readme.md

```diff
@@ -68,39 +75,59 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT SELECT ID, DESCRIPTION FROM SPENDS
+WHERE LOWER(DESCRIPTION) like '%fee%'
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT SELECT 
+    expense_area
+FROM 
+    expense_areas
+WHERE  lower(expense_area)
+like  '%food'
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE SELECT
+    TO_CHAR(date, 'MONTH') as MONTH, SUM(amount)
+FROM
+    spends
+GROUP BY TO_CHAR(date, 'MONTH')
+ORDER BY SUM(amount) 
 ```
 
 **Farnoosh:** Thanks, that's really useful. We also need to know the total amount spent on each supplier. Can you help us with that?
 
 **You:** Sure thing. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT 
+select 
+    SUM(amount) 
+from 
+    spends;
```

Comment:
look at doing this by each supplier :) 

Try to return a table which has the suppliers and the total amount for each